apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -91,13 +91,21 @@ export const styles = {
 const SectionsTable = React.createClass({
   propTypes: {
     sections: shapes.sections,
-    isRtl: React.PropTypes.bool.isRequired,
+    isRtl: PropTypes.bool.isRequired,
     isTeacher: PropTypes.bool.isRequired,
-    canLeave: PropTypes.bool.isRequired
+    canLeave: PropTypes.bool.isRequired,
+    updateSections: PropTypes.func.isRequired
   },
 
-  onLeave() {
-    //TODO: Clicking this button should remove the student from that section. Remove display: 'none' from leaveCol styles so the button will appear for students who have permission to leave sections.
+  onLeave(sectionCode) {
+    $.post('/followers/remove_async', {section_code: sectionCode})
+      .done(function (data) {
+        console.log(data.sections);
+        this.props.updateSections(data.sections);
+      }.bind(this))
+      .fail(function () {
+      }.bind(this)
+    );
```

Comment:
Same comment about arrow functions.

Also, did you mean to leave in that console.log on line 103?